readthedocs/search/parsers.py

```diff
@@ -435,34 +472,19 @@ def _process_fjson(self, fjson_path):
                 try:
                     # Create a new html object, since the previous one could have been modified.
                     body = HTMLParser(data["body"])
-                    domain_data = self._generate_domains_data(body)
```

Comment:
doesn't look like we need this whole try block anymore.

Reply:
The whole block in fact!

Reply:
612c68772